spanner/ddl/ast.go

```diff
@@ -200,18 +267,28 @@ func (ct CreateTable) PrintCreateTable(config Config) string {
 			cols += strings.Repeat(" ", n-len(c)) + " -- " + colComment[i]
 		}
 	}
+
 	for _, p := range ct.Pks {
 		keys = append(keys, p.PrintIndexKey(config))
 	}
 	var tableComment string
 	if config.Comments && len(ct.Comment) > 0 {
 		tableComment = "--\n-- " + ct.Comment + "\n--\n"
 	}
+
 	var interleave string
 	if ct.Parent != "" {
-		interleave = ",\nINTERLEAVE IN PARENT " + config.quote(ct.Parent)
+		if config.TargetDb == constants.TARGET_EXPERIMENTAL_POSTGRES {
+			interleave = " INTERLEAVE IN PARENT " + config.quote(ct.Parent)
+		} else {
+			interleave = ",\nINTERLEAVE IN PARENT " + config.quote(ct.Parent)
```

Comment:
Do we need separate handling for pg and normal spanner? In PrintCreateTable just above, it seems the ',' was added at the end of stmt for both normal and pg spanner.